Cython/Distutils/build_ext.py

```diff
@@ -12,14 +12,32 @@
 
 
 class new_build_ext(_build_ext, object):
-    def finalize_options(self):
-        if self.distribution.ext_modules:
-            nthreads = getattr(self, 'parallel', None)  # -j option in Py3.5+
-            nthreads = int(nthreads) if nthreads else None
-            from Cython.Build.Dependencies import cythonize
-            self.distribution.ext_modules[:] = cythonize(
-                self.distribution.ext_modules, nthreads=nthreads, force=self.force)
```

Comment:
We can remove `nthreads` since it is already paralelized here:
https://github.com/python/cpython/blob/bf1812ef61fe95ae96fbcebd6e384afde57f2790/Lib/distutils/command/build_ext.py#L451

Reply:
But that will not parallelise the Cython runs. It only applies to the C compiler runs later.

Reply:
If I am looking correctly to the code, it is. The PR is moving running cython to the `def build_extension()` method and it is run in parralel here:
 
https://github.com/python/cpython/blob/bf1812ef61fe95ae96fbcebd6e384afde57f2790/Lib/distutils/command/build_ext.py#L465